src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -52,6 +52,79 @@ using namespace orion;
 
 
 
+typedef struct CompoundSave
```

Comment:
If at the end we go with the workaround implemented by this PR, an explanation of the workaround should be included in a comment in the code in order to guide for a definitive solution. The text in the PR body is a good starting point for that.

Issue #3162 would remain open until the definitive solution comes.